data/json/requirements/materials.json

```diff
@@ -12,6 +12,15 @@
     "components": [ [ [ "lead", 1 ], [ "bismuth", 1 ], [ "silver_small", 1 ], [ "gold_small", 1 ] ] ]
   },
   {
+    "id": "ammo_bullet",
```

Comment:
On the other hand, the standarization I had decided on started by working off of a volume comparison, not a weight comparison. If I do migrate and standardize small copper, it will only be the weight I intend to fix. The "generic bullet lead" component already ignores the densities of material anyway, and making bullet material demands completely conserve mass is a much larger undertaking.

Simply removing the added requirement template for copper jacketing is perhaps the only required step, and my planned changes should not require closing this unless a full conservation-of-mass pull request is in the works.

Balancing the copper demand to be more tolerable for players with is presently the primary objective of this pull request.